### PR TITLE
Added the TTL Config value for the heartbeat plugin.

### PR DESCRIPTION
**Pull request by indualagarsamy**
Changed the xml to snippets instead.

Relates to https://github.com/Particular/ServiceControl.Plugin.Nsb3.Heartbeat/issues/2
Relates to https://github.com/Particular/ServiceControl.Plugin.Nsb4.Heartbeat/issues/2
Relates to https://github.com/Particular/ServiceControl.Plugin.Nsb5.Heartbeat/issues/3

### CODE REVIEW

File: servicecontrol/plugins/heartbeat.md [hunk 1]
```diff
@@ -44,12 +44,19 @@ If you are using the older version of the plugin, namely **ServiceControl.Plugin
...
 
 Where the value is convertible to a `TimeSpan` value. In the above sample you are setting the endpoint heartbeat interval to 40 seconds.
 
 When configuring heartbeat interval, make sure Service Control setting [`HeartbeatGracePeriod`](/servicecontrol/creating-config-file.md#configuration-options-servicecontrol-heartbeatgraceperiod) is greater than the heartbeat interval.
 
+
+### TTL
+When the plugin sends heartbeat messages, the default TTL is fixed to four times the configured value of the Heartbeat interval. In some cases, this still caused the heartbeat message to end up in the Dead Letter Queue depending on the message load in the ServiceControl Queue.
+
+TTL is now configurable, same as the heartbeat interval.
```
Comment (SimonCropp) [inline servicecontrol/plugins/heartbeat.md]: "Is now" is this missing some version info?

Also if this is meant ti be a paragraph then please add anothe newline at the end

File: servicecontrol/plugins/heartbeat.md [hunk 2]
```diff
@@ -44,12 +44,19 @@ If you are using the older version of the plugin, namely **ServiceControl.Plugin
...
+Add the app setting in app.config as shown for to configure the TTL to a custom value instead of the default value based on heartbeat interval. Provide the timespan string for the value as shown. In this example, a heartbeat message will be sent every 30 seconds and the TTL for the heartbeat message is 3 minutes.
```
Comment (SimonCropp) [inline servicecontrol/plugins/heartbeat.md]: "For to configure"

File: servicecontrol/plugins/heartbeat.md [hunk 3]
```diff
@@ -44,12 +44,19 @@ If you are using the older version of the plugin, namely **ServiceControl.Plugin
...
-```
+<!-- import heartbeatsIntervalConfig -->
 
 Where the value is convertible to a `TimeSpan` value. In the above sample you are setting the endpoint heartbeat interval to 40 seconds.
 
 When configuring heartbeat interval, make sure Service Control setting [`HeartbeatGracePeriod`](/servicecontrol/creating-config-file.md#configuration-options-servicecontrol-heartbeatgraceperiod) is greater than the heartbeat interval.
 
+
+### TTL
+When the plugin sends heartbeat messages, the default TTL is fixed to four times the configured value of the Heartbeat interval. In some cases, this still caused the heartbeat message to end up in the Dead Letter Queue depending on the message load in the ServiceControl Queue.
```
Comment (SimonCropp) [inline servicecontrol/plugins/heartbeat.md]: The content about the dead letter queue does not belong here. That problem is not specific to heardbeats. Also dlq is specific to soecific transports and hearbeats operates several levels above that

**Comment by indualagarsamy** [comment]
@SimonCropp - I've made the suggested changes.

**Pull request merged**